packages/cli-kit/src/ui.ts

```diff
@@ -48,6 +48,31 @@ interface BaseQuestion<TName extends string> {
   result?: (value: string) => string | boolean
 }
 
+export interface SimpleTaskOptions {
+  title: string
+  success?: string
+  task: () => Promise<void | string>
```

Comment:
It's hard to tell what `string` means without reading at the docs or the implementation. What about using a custom type:

```ts
Promise<void | { successMessage: string }>
```

Reply:
Also, being able to pass a success message through the return value of the `task` function and the `success?` attribute seems redundant. Wouldn't it be sufficient with the return value of the task?